spec/features/publish/courses/editing_subject_spec.rb

```diff
@@ -13,30 +13,69 @@
     when_i_select_a_primary_subject('Primary with English')
     and_i_click_continue
     then_i_am_met_with_course_details_page
-    and_i_should_see_a_success_message
+    and_i_should_see_subject_updated_success_message
```

Comment:
I can see we have covered the further education scenarios in the unit testing -- are there features tests that cover that journey? (they might exist, but just haven't been changed, so I don't see them in this PR). Would be good to add a create / update scenario with FE if they don't already exist. That way, if we ever find ourselves in a position to do some refactoring, we can be confident we haven't broken any essential journeys.